joplin/templates/wagtailadmin/shared/field.html

```diff
@@ -17,5 +17,15 @@
         {% if show_help_text|default_if_none:True and field.help_text %}
             <p class="help">{{ field.help_text }}</p>
         {% endif %}
+        {% comment %} uncomment this stuff to add errors back to fields,
```

Comment:
@Niicck this is why you probably weren't seeing validation errors pop up in Joplin